tasks/01_cases/clean.py

```diff
@@ -6,7 +6,7 @@
 if __name__ == '__main__':
     """Drop cases table."""
 
-    query = 'DROP TABLE IF EXISTS cases'
+    query = 'DROP TABLE IF EXISTS cases CASCADE'
```

Comment:
To keep things from having unexpected consequences, this query should only drop the cases table instead of cascading to dependent tables -- the others (allegations, etc) should be dropped by their own task's `clean` scripts.

Reply:
if i recall correctly, the `clean` on this task errored out on me because of those other dependencies. will take this out.